config/config.json.in

```diff
@@ -26,6 +26,12 @@
       "fetch_page_fault": true,
       "software_check": true,
       "reserved_exceptions": false
+    },
+    "reserved_behaviour": {
+      // The configuration option determines how to handle the reserved behaviour `amocas_odd_registers`.
+      // "ignore"  – ignore the condition and continue executing the AMO instruction.
```

Comment:
But what happens in this case when you access `x31`? Looking at the code it will wrap to `(x31, x0)`. Is anyone actually going to implement this?

What's the motivation for supporting this reserved behaviour?